test/visual-diff/visual-tests

```diff
@@ -605,6 +605,15 @@
       ],
       "interactive_tests": "examples/visual-tests/amp-story/amp-story-tooltip-desktop.js"
     },
+    {
+      "url": "examples/visual-tests/amp-story/amp-story-affiliate-link.html",
+      "name": "amp-story: affiliate link",
+      "viewport": {"width": 320, "height": 480},
```

Comment:
Could you also add one in desktop view please?

Reply:
If we don't expect the output to look different on mobile or desktop, it might be OK to skip desktop tests to keep our number of screenshots "low". I don't feel super strongly here though, but that might just be "more screenshots to approve" every time we make a chance.

Reply:
I think it's okay having a mobile and desktop screenshots. There's one difference - the `:hover` style that is different for touchscreens, which I account for here.
https://github.com/ampproject/amphtml/blob/97fd8a74167a2ecd7e202db8fb2cbcc7392b112e/extensions/amp-story/1.0/amp-story-affiliate-link.css#L135-L144